.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build And Formatting CI
```

Comment:
Try to name this something shorter. We can then get a nice badge for the readme indicating whether this CI check is passing or not.

Example: ![](https://github.com/hazel-ui/hazel-ui/workflows/build/badge.svg)

The `name` here will be the left label of the badge.